python/cudf/cudf/tests/test_timedelta.py

```diff
@@ -809,6 +817,7 @@ def test_timedelta_datetime_index_ops_misc(
         ),
     ],
 )
+@pytest.mark.filterwarnings("ignore::RuntimeWarning")
```

Comment:
We can be more specific:
```suggestion
@pytest.mark.filterwarnings("ignore:divide by zero:RuntimeWarning:pandas")
```

Syntax comes from here:
> Individual warnings filters are specified as a sequence of fields separated by colons:
> `action:message:category:module:line`
https://docs.python.org/3/library/warnings.html#describing-warning-filters